R/foverlaps.R

```diff
@@ -15,7 +15,7 @@ foverlaps <- function(x, y, by.x = if (!is.null(key(x))) key(x) else key(y), by.
   mult = match.arg(mult)
   if (type == "equal")
     stop("type = 'equal' is not implemented yet. But note that this is just the same as a normal data.table join y[x, ...], unless you are also interested in setting 'minoverlap / maxgap' arguments. But those arguments are not implemented yet as well.")
-  if (maxgap > 0L || minoverlap > 1L)
+  if (maxgap != 0L || minoverlap != 1L)
```

Comment:
maxgap < 0 and minoverlap < 1 both are checked in lines 6 and 8. What's the purpose of this change?